signature/src/policy.rs

```diff
@@ -0,0 +1,460 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use oci_distribution::Reference;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::convert::TryFrom;
+use std::fs;
+use std::vec::Vec;
+use std::str::FromStr;
+
+use crate::image;
+use crate::signatures;
+
+#[derive(EnumString, Display, Debug, PartialEq)]
+pub enum PolicyReqType {
+    #[strum(to_string = "insecureAcceptAnything")]
+    TypeAccept,
+    #[strum(to_string = "reject")]
+    TypeReject,
+    #[strum(to_string = "signedBy")]
+    TypeSignedBy,
+}
+
+#[derive(EnumString, Display, Debug, PartialEq)]
+pub enum KeyType {
+    #[strum(to_string = "GPGKeys")]
```

Comment:
Maybe it should be "GPGKey" (singular)?

Reply:
This actually refers to GPG public key ring, so `GPGKeys` is more appropriate.